FoundationsMastery/Kitten.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FoundationsMastery
+{
+    public class Kitten : Cat
+    {
+        public override void Meow()
+        {
+            base.Meow();
```

Comment:
This needs to do something different than what the parent does. `base.Meow()` explicitly calls the parent's `Meow` method.